Sources/UI/Views/PollQuestionView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  PollQuestionView.swift
+//  Rover
+//
+//  Created by Andrew Clunis on 2019-07-09.
+//  Copyright © 2019 Rover Labs Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class PollQuestionView: UIView {
```

Comment:
It seems this is a regular UIView which embeds a UITextView and pins all the edges. Perhaps it could just extend from UITextView directly?

Reply:
Because I'm using `configureContent` to set the insets, which are done with constraints.

If I want to drop the containing UIView I would have to adopt another code path. Which, to be fair, might be as simple as using `UIView.layoutMargins` instead of constraints.  What do you think?

Reply:
I see that but from what I can tell, there are no insets..  you're passing a hardcoded value of `.zero` which just pins the edges exactly to each other which, unless I'm missing something, makes the subview useless?